src/components/ec-container/ec-container.vue

```diff
@@ -30,6 +30,7 @@ export default {
   flex-direction: row;
   min-height: 100vh;
   align-items: stretch;
+  position: relative;
```

Comment:
This is a "logic" required by ec-panel, it should not be in ec-container. You are tightly coupling both components together and you are saying ec-panel must be always a child of the ec-container. I'd rather avoid that.
I would remove this logic and move it to EBO. Add CSS class to the `ec-container` in EBO and add the `position: relative` rule.